modules/security-context-constraints-command-reference.adoc

```diff
@@ -3,9 +3,9 @@
 // * authentication/managing-security-context-constraints.adoc
 
 [id="security-context-constraints-command-reference_{context}"]
-= Security context constraints reference commands
+= Reference of security context constraints commands
 
-You can manage SCCs in your instance as normal API objects using the CLI.
+You can manage security context constraints (SCCs) in your instance as normal API objects using the OpenShift CLI (`oc`).
 
 ifdef::openshift-enterprise,openshift-webscale,openshift-origin[]
 [NOTE]
```

Comment:
Below IMPORTANT statement is also declared in 'Default security context constraints' section, is it needed here again?

Reply:
I was going to move it down to the section on updating SCCs, but it looks like there is already a similar warning there too about not editing settings on the default SCCs. So I've removed this.